src/darshana/ArmStrongNum.java

```diff
@@ -0,0 +1,32 @@
+package darshana;
+
+import java.util.Scanner;
+
+public class ArmStrongNum
+{
+    public static void main(String args[])
+    {
+        int n, nu, num=0, rem;
+        Scanner scan = new Scanner(System.in);
+		
+        System.out.print("Enter any Positive Number : ");
+        n = scan.nextInt();
+		
+        nu = n;
+		
+        while(nu != 0)
```

Comment:
make separate method for logic always, this is will be easy to understand & gives better readability.